Makefile

```diff
@@ -6,23 +6,30 @@
 # To install the build kernel modules: run (as root) `make modules_install`
 ###########################################################################
 
-include utils.mk
+###########################################################################
+# variables
+###########################################################################
 
-all: modules
+include utils.mk
 
```

Comment:
Inclusion of utils.mk isn't really a variable.  Would it make sense to move this include above the `variables` comment?

Reply:
good point, let me fix it